styles/components/page-header.css

```diff
@@ -4,3 +4,51 @@
   align-items: center;
   min-width: 336px;
 }
+
+.navigation__list {
```

Comment:
As the filename is page-header, this should only contain classes starting with `page-header`. One file per component, please.